src/home/views/index.tsx

```diff
@@ -8,57 +9,64 @@ import { orderListUrl } from "../../orders/urls";
 import { productListUrl, productVariantEditUrl } from "../../products/urls";
 import { OrderStatusFilter, StockAvailability } from "../../types/globalTypes";
 import HomePage from "../components/HomePage";
-import { HomePageQuery } from "../queries";
+import { useHomePage } from "../queries";
 
 const HomeSection = () => {
   const navigate = useNavigator();
   const { user } = useUser();
   const { channel } = useAppChannel();
 
+  const noChannel = !channel && typeof channel !== "undefined";
+
+  const { data } = !noChannel
+    ? useHomePage({
+        displayLoader: true,
+        variables: { channel: channel?.slug }
+      })
+    : { data: null };
+
   return (
-    <HomePageQuery displayLoader variables={{ channel: channel.slug }}>
-      {({ data }) => (
-        <HomePage
-          activities={maybe(() =>
-            data.activities.edges.map(edge => edge.node).reverse()
-          )}
-          orders={maybe(() => data.ordersToday.totalCount)}
-          sales={maybe(() => data.salesToday.gross)}
-          topProducts={maybe(() =>
-            data.productTopToday.edges.map(edge => edge.node)
-          )}
-          onProductClick={(productId, variantId) =>
-            navigate(productVariantEditUrl(productId, variantId))
-          }
-          onOrdersToCaptureClick={() =>
-            navigate(
-              orderListUrl({
-                status: [OrderStatusFilter.READY_TO_CAPTURE]
-              })
-            )
-          }
-          onOrdersToFulfillClick={() =>
-            navigate(
-              orderListUrl({
-                status: [OrderStatusFilter.READY_TO_FULFILL]
-              })
-            )
-          }
-          onProductsOutOfStockClick={() =>
-            navigate(
-              productListUrl({
-                stockStatus: StockAvailability.OUT_OF_STOCK
-              })
-            )
-          }
-          ordersToCapture={maybe(() => data.ordersToCapture.totalCount)}
-          ordersToFulfill={maybe(() => data.ordersToFulfill.totalCount)}
-          productsOutOfStock={maybe(() => data.productsOutOfStock.totalCount)}
-          userName={getUserName(user, true)}
-          userPermissions={user?.userPermissions || []}
-        />
-      )}
-    </HomePageQuery>
+    <HomePage
+      activities={data?.activities.edges.map(edge => edge.node).reverse()}
+      orders={!noChannel ? data?.ordersToday.totalCount : 0}
```

Comment:
I think in this case HomePage component should have a default when orders is undefined. Then it also doesn't require to check if `!noChannel`